onnxruntime/core/providers/tvm/tvm_api.h

```diff
@@ -4,26 +4,35 @@
 #ifndef TVM_API_H
 #define TVM_API_H
 
+#include <vector>
+#include <string>
+
 #include "tvm_common.h"
+#include "tvm_defaults.h"
 
 namespace onnxruntime {
 namespace tvm {
     TvmModule TVMCompile(const std::string& onnx_txt,
-                                    const std::string& model_path,
-                                    const std::string& target,
-                                    const std::string& target_host,
-                                    int opt_level,
-                                    int opset,
-                                    bool freeze_params,
-                                    const std::vector<std::vector<int64_t>>& input_shapes,
-                                    bool nhwc = false,
-                                    const std::string& tuning_logfile = "",
-                                    const std::string& tuning_type = "AutoTVM");
+                         const std::string& model_path,
+                         const std::string& executor,
+                         const std::string& target,
+                         const std::string& target_host,
+                         int opt_level,
+                         int opset,
+                         bool freeze_params,
+                         const std::vector<std::vector<int64_t>>& input_shapes,
+                         bool nhwc = false,
+                         const std::string& tuning_logfile = "",
+                         const std::string& tuning_type = std::string(onnxruntime::tvm::default_tuning_type));
     void TVMSetInputs(TvmModule& mod, std::vector<size_t>& inds, std::vector<DLTensor>& inputs);
+    void TVM_VM_SetInputs(TvmModule& mod, std::vector<DLTensor>& inputs);
+    void TVMGetOutputs(TvmModule& mod, std::vector<DLTensor>& outputs);
+    void TVMGet_VM_Outputs(TvmModule& mod, std::vector<DLTensor>& outputs);
```

Comment:
We should name these functions either `TVM_VM_SetInputs` and `TVM_VM_GetInputs` or `TVMSet_VM_Inputs` and `TVMGet_VM_Inputs` for consistency.

Reply:
Thanks! done